configs/sites/tier1/acorn/compilers.yaml

```diff
@@ -6,18 +6,17 @@ compilers:
       cxx: CC
       f77: ftn
       fc: ftn
-    flags: {}
+    flags:
+      cflags: -static-libgcc -static-libstdc++ -Bstatic -lstdc++ -Bdynamic -lm -lpthread
```

Comment:
For my curiosity, why is this needed?

Reply:
There have been issues on Acorn in the past with libstdc++ changing or being inconsistent between login and compute nodes in ways that cause dynamic library loading problems, so basically it's a small risk reducer (and so far everything I've tested compiles fine with it). The 'redundant' -static-libstdc++ and `-Bstatic -lstdc++` are because of some very strange quirk with the Cray wrappers and this was the only way to actually force it to use the static libstdc++ :upside_down_face: